configure_gpu_ubuntu20.md

```diff
@@ -0,0 +1,65 @@
+# Configuration Guide for the Intel® Graphics Compute Runtime for OpenCL™ on Ubuntu* 20.04
+
+Intel® Graphics Compute Runtime for OpenCL™ driver components required to use the GPU plugin and write custom layers for Intel® Integrated Graphics. 
+The driver is installed in the OpenVINO™ Docker image but you need to do some configuration steps to activate it in the container for a non-root user if you have Ubuntu 20.04 on your host.
+In order to access GPU capabilities, a user needs to have the correct permissions on host and Docker container. 
+The following will list the group assigned ownership of the render nodes on your host:
+```bash
+$ stat -c "group_name=%G group_id=%g" /dev/dri/render*
+group_name=render group_id=134
+```
+OpenVINO™ Docker images don't contain render group for openvino (non-root) user, because render group doesn't have strict group ID unlike video group. 
+We suppose two options to configure a container for you. Please choose one:
+
+
+# 1. Configure a container to use GPU device on Ubuntu 20 host under non-root user
+To run a OpenVINO container as non-root with access to GPU device, please specify render group ID from your host:
+```bash
+$ docker run -it --rm --device /dev/dri  --group-add=<render_group_id_on_host> <image_name> 
```

Comment:
what is `render_group_id_on_host`? I think It is not clear for the user

Reply:
I think I explain it in the next message -> For example, get render group ID on your host